pkgs/development/python-modules/bx-python/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, buildPythonPackage, numpy, cython, zlib, six, python-lzo
+, nose }:
+
+buildPythonPackage rec {
+  pname = "bx-python";
+  version = "0.8.6";
+
+  src = fetchFromGitHub {
+    owner = "bxlab";
+    repo = "bx-python";
+    rev = "v${version}";
+    sha256 = "1i40vmn8n83vqcpqj843riv9vp16s753jc4wc90p0cmrnhmzcv13";
+  };
+
+  buildInputs = [ zlib cython ];
```

Comment:
```suggestion
  nativeBuildInputs = [ cython ];
  buildInputs = [ zlib cython ];
```

Reply:
Thanks for the review, fixed this and the commit messages.